lib/functoria/cli.mli

```diff
@@ -140,4 +140,5 @@ type 'a result =
     also carry global command-line parameters. *)
 
 val peek : ?with_setup:bool -> mname:string -> string array -> unit result
-(** [peek] is the same as {!val:eval} but without failing on unknown arguments. *)
+(** [peek] is the same as {!val:eval} but without failing on unknown arguments.
+*)
```

Comment:
is this due to a bug in accounting for 80-chars wide lines?

Reply:
Yes, the `*)` was previously breaking the margin.
By aligned with the opening parenthesis, do you mean this ?
```
(** ...
 *)
```
This strange formatting only appears when the comment is just long enough to hit the margin but do not start to wrap.
In other cases, the `*)` is placed at the end of the last paragraph:
```
(** Every other comments
    are like this *)
```

Reply:
The formatting of regular comments is slightly different, if the `*)` exceed the margin, the last word of the paragraph wraps with it:
```ocaml
(* *[peek] is the same as {!val:eval} but without failing on unknown
   arguments. *)
```
Do you think that's better than breaking only the `*)` ?